sales.html

```diff
@@ -6,9 +6,24 @@
     <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <section id ="cookieTableSection">
+    <section id="cookie_table_section">
     </section>
-
+    <div class="form">
+      <form id="new-store-form">
+        <fieldset>
+          <legend> Enter New Cookie Info </legend>
+          <label for="cookieStoreName">Name of Store:</label>
```

Comment:
This is great! That said, technically the [for attribute points at the id of the labelled form element, not the name](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label).